mods/cnc/maps/gdi05b/map.yaml

```diff
@@ -0,0 +1,820 @@
+		Location: 49,17
+		Owner: Nod
+	Fact1: fact
+		Location: 51,17
+		Owner: Nod
+	GdiNuke1: nuke
+		Location: 33,51
+		Owner: AbandonedBase
+		Health: 0.4570313
+	GdiProc1: proc
+		Location: 27,52
+		Owner: AbandonedBase
+		Health: 0.4765625
+		FreeActor: False
+	GdiHarv: harv
+		Location: 27,55
+		Owner: AbandonedBase
+		Facing: 64
+	GdiWeap1: weap
+		Location: 35,51
+		Owner: AbandonedBase
+		Health: 0.4140625
+	GdiNuke2: nuke
+		Location: 31,51
+		Owner: AbandonedBase
+		Health: 0.3945313
+	GdiPyle1: pyle
+		Location: 31,54
+		Owner: AbandonedBase
+		Health: 0.4570313
+	GdiSilo1: silo
+		Location: 26,51
+		Owner: AbandonedBase
+		Health: 0.40625
+	GdiSilo2: silo
+		Location: 29,51
+		Owner: AbandonedBase
+	Sam1: sam
+		Location: 52,25
+		Owner: Nod
+	Sam2: sam
+		Location: 38,25
+		Owner: Nod
+	Sam3: sam
+		Location: 40,17
+		Owner: Nod
+	Sam4: sam
+		Location: 26,37
+		Owner: Nod
+
+Smudges:
+	sc3 15,56 0:
+	sc2 40,53 0:
+	sc2 24,53 0:
+	sc5 39,52 0:
+	sc1 24,52 0:
+	sc4 39,51 0:
+
+Rules:
+	Player:
+		-ConquestVictoryConditions:
+		MissionObjectives:
+			EarlyGameOver: true
+		EnemyWatcher:
+	World:
+		-CrateSpawner:
+		-SpawnMPUnits:
+		-MPStartLocations:
+		ObjectivesPanel:
+			PanelName: MISSION_OBJECTIVES
+		LuaScript:
+			Scripts: gdi05b.lua
+	^Vehicle:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Tank:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Helicopter:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Infantry:
+		SpawnViceroid:
+			Probability: 0
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Plane:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Ship:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Building:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+		AnnounceOnSeen:
+	^Wall:
+		Tooltip:
+			ShowOwnerRow: false
+	^Husk:
+		Tooltip:
+			GenericVisibility: Enemy, Ally, Neutral
+			GenericStancePrefix: false
+			ShowOwnerRow: false
+	#### STRUCTS ####
+	SBAG:
+		Buildable:
+			Prerequisites: ~disabled
+	GTWR:
+		Buildable:
+			Prerequisites: ~disabled
+	ATWR:
+		Buildable:
+			Prerequisites: ~disabled
+	WEAP:
+		Buildable:
+			Prerequisites: ~disabled
+	CYCL:
+		Buildable:
+			Prerequisites: ~disabled
+	NUK2:
+		Buildable:
+			Prerequisites: ~disabled
+	FIX:
+		Buildable:
+			Prerequisites: ~disabled
+	HPAD:
+		Buildable:
+			Prerequisites: ~disabled
+	BRIK:
+		Buildable:
+			Prerequisites: ~disabled
+	EYE:
+		Buildable:
+			Prerequisites: ~disabled
+	#### NOD ####
+	GUN:
+		Buildable:
+			Prerequisites: ~disabled	
+	OBLI:
+		Buildable:
+			Prerequisites: ~disabled	
+	TMPL:
+		Buildable:
+			Prerequisites: ~disabled	
+	#### UNITS ####
+	E2:
+		Buildable:
+			Prerequisites: ~pyle
+	E3:
+		Buildable:
+			Prerequisites: ~disabled
+	HARV:
+		Buildable:
+			Prerequisites: ~disabled
+	MTNK:
+		Buildable:
+			Prerequisites: ~disabled
+	HTNK:
+		Buildable:
+			Prerequisites: ~disabled
+	TRAN:
```

Comment:
You don't need to disable `TRAN`, `ORCA` and `HELI`, as the player is not able to get a `HPAD` (is he?).